DynamicProgramming/c++/longestCommonSubsequence.cpp

```diff
@@ -1,4 +1,4 @@
-#include <bits/stdc++.h>
+#include <bits/stdc++.h> // including every standard library.

```

Comment:
This header is quite linux specific and only support on the ```gcc```. Additional it increase the compile time. Remove it!